ztp/resource-generator/Containerfile

```diff
@@ -15,17 +15,27 @@ WORKDIR $PGT_ROOT
 RUN make build
 WORKDIR $SC_ROOT
 RUN make build
+WORKDIR /go/src/
+RUN curl -L $(curl https://api.github.com/repos/stolostron/policy-generator-plugin/releases/latest | jq -r '.tarball_url') -o policy-generator-plugin.tar.gz
+RUN mkdir policy-generator-plugin && tar -xzvf policy-generator-plugin.tar.gz -C policy-generator-plugin --strip-components=1
```

Comment:
Should we have a more explicit versioning here other than "latest". For example, should the 4.11 version of the ztp-site-generator container always pull (example) version 1.2 (no z-stream listed so the latest from 1.2) of stolostron. Then when a new y-stream version comes out there is an explicit switch up to 1.3, etc? This way the dependency between ztp-site-generate container and underlying stolostron/acm version is explicit (and can be tested).